File: Python/BOJ_2445_별 찍기 - 8.py
import sys

num = int(sys.stdin.readline())
for i in range(1, num + 1):
    print('*' * i, end = '')
    print(' ' * ((num - i) * 2), end = '')
    print('*' * i)

for j in range(num - 1, 0, -1):
    print('*' * j, end = '')
    print(' ' * ((num - j) * 2), end = '')
    print('*' * j)

File: C/BOJ_2445_별 찍기 - 8.c
# include <stdio.h>

int main()
{
	int num;
	scanf("%d", &num);
	for (int i = 0; i < num; i++) {
		for (int j = 0; j < i + 1; j++) printf("*");
		for (int k = 2; k < (num - i) * 2; k++) printf(" ");
		for (int l = 0; l < i + 1; l++) printf("*");
		printf("\n");
	}
	for (int i = 0; i < num; i++) {
		for (int j = 1; j < num - i; j++) printf("*");
		for (int k = 0; k < (i * 2) + 2; k++) printf(" ");
		for (int l = 1; l < num - i; l++) printf("*");
		printf("\n");
	}
	return 0;
}